content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Announcing Istio 1.9.0
+linktitle: 1.9
+subtitle: Major Update
+description: Istio 1.9 release announcement.
+publishdate: 2021-02-09
+release: 1.9.0
+skip_list: true
+aliases:
+    - /news/announcing-1.9
+    - /news/announcing-1.9.0
+---
+
+We are pleased to announce the release of Istio 1.9.
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** `istioctl install` will detect different Istio version installed (istioctl, control plan version) and display warning.
+  ([Issue #18487](https://github.com/istio/istio/issues/18487))
+
+- **Added** post-install/in-place upgrade verification of control plane health. Use `--verify` flag with `istioctl install` or `istioctl upgrade`  ([Issue #21715](https://github.com/istio/istio/issues/21715))
+
+- **Added** Add pprof endpoint to pilot-agent.
```

Comment:
add a link to pprof?  https://github.com/google/pprof

Reply:
Done.